playbooks/meta/livy-spark3.yml

```diff
@@ -2,13 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
-- ansible.builtin.import_playbook: ../livy-spark3_server_install.yml
-- ansible.builtin.import_playbook: ../livy-spark3_kerberos_install.yml
-- ansible.builtin.import_playbook: ../livy-spark3_ssl-tls_install.yml
+- import_playbook: ../livy-spark3_jmx-exporter_config.yml
```

Comment:
Can you update to use FQCN ? I know that this was generated by `tdp playbooks`. We should update the `ŧdp` command.

Reply:
FQCN updated. 
Note that `ansible-lint` also has the following complains : 
~~~
name[play]: All plays should be named.  
~~~